lib/mixlib/install/generator/powershell/scripts/helpers.ps1.erb

```diff
@@ -150,7 +146,7 @@ function Custom-GetFileHash ($Path, $Algorithm) {
 }
 
 function Get-SHA256Converter {
-  if (Is-FIPS) {
+  if ($(Is-FIPS) -ge 1) {
     New-Object -TypeName Security.Cryptography.SHA256Cng
```

Comment:
so sorry not to be mentioning this earlier but I'm also wondering if we can consolidate here and just use `SHA256Cng` on fips and non fips. Not sure if we could use it on core too and I wouldn't worry about that. If we could, we wouldn't need a separate code path for fips I don't think.